index.html

```diff
@@ -47,6 +47,11 @@ <h1>The Regis Altare Birthday Project 2023</h1>
         >
       </footer>
     </section>
+    <section id="card-list">
+      <!--Test image serving-->
+      <img class="card-image" src="images/elysium_delivery_services.jpg">
```

Comment:
I checked the deploy preview, and it indeed converted this into a Cloudinary URL :tada: 
So my understanding is if this image were hypothetically not on the home page but on some other page, there would be no network bandwidth consumed when the home page loads in?
In contrast, if we did not use the cloudinary-netlify plugin, this image would use network bandwidth - even though it's not immediately needed for the current page - because it's a static image hardcoded into our website's files.
Is that right?